examples/with-backup/README.md

```diff
@@ -0,0 +1,82 @@
+growi-docker-compose with backup example
+========================================
+
+# Introduction
+
+Growi has MongoDB to store own pages data and users data.
+
+This guide show how to backup MongoDB in case of using [weseek/growi-docker-compose](https://github.com/weseek/growi-docker-compose).
+
+# How to backup with "weseek/mongodb-awesome-backup"
+
+We show how to backup in case of you use Growi in docker-compose environment.
+
+## Requirements
+
+- AWS S3 buckets and IAM account to store backup files
+
+## Quick start
+
+1. Override `docker-compose.override.yml` to your `docker-compose.yml`
+2. Create file `.env/SOME_FILE_NAME` or copy `.env/s3-secret-keys`, and set environment about secret ID and access key of S3.
+    ```
+    AWS_ACCESS_KEY_ID=${Your IAM Access Key ID}
+    AWS_SECRET_ACCESS_KEY=${Your IAM Secret Access Key}
+    ```
+3. Copy config file of crond `./crontab/root`, and specify timing when execute mongodb-awesome-backup.
+4. Set file permission `./crontab/root` to `root:root`.
+5. Edit `docker-compose.yml` these values
+    - Set `S3_TARGET_BUCKET_URL` to valid URL of S3 bucket.
+    - Set `SOME_FILE_NAME` to valid path of environment file you create.
+
+Then, you can get backup of MongoDB in S3.<br>
+The backup container is already execute with Growi when `docker-compose up`.
+
+
+## Configurations
+
+You can use these environment values in `environment` section in docker-compose.yml.
+
+- AWS_ACCESS_KEY_ID
+- AWS_SECRET_ACCESS_KEY
+- S3_TARGET_BUCKET_URL
+- BACKUPFILE_PREFIX
+- MONGODB_HOST
+- MONGODB_DBNAME
+- MONGODB_USERNAME
+- MONGODB_PASSWORD
+- MONGODB_AUTHDB
+
+For detail information, see the original site [weseek/mongodb-awesome-backup](https://github.com/weseek/mongodb-awesome-backup).
+
+## Optional settings
+
+### Not if you want create environment file, you should to specify these in `docker-compose.yml`.
+
+```yaml:docker-compose.yml
+    : <snip>
+  backup:
+    image: weseek/mongodb-awesome-backup:0.2.0
+    environment:
+      - CRONMODE=true
+      - S3_TARGET_BUCKET_URL=s3://${REPLACE_THIS_TO_YOUR_BUCKET_NAME}/
+      - AWS_ACCESS_KEY_ID=${Your IAM Access Key ID}
+      - AWS_SECRET_ACCESS_KEY=${Your IAM Secret Access Key}
+    links:
+      - mongo:mongo
+    volumes:
+      - ./crontab/root:/var/spool/cron/crontabs/root
+    : <snip>
+```
+
+# Operation log of backup
+
+Logging of "mongodb-awesome-backup" is output STDOUT of container.
+
+So you can see log by `docker logs` command.
+
+# Note
+
+"mongodb-awesome-backup" is backup container.<br>
```

Comment:
Don't use `<br>` in markdown. Use a couple of spaces.

Reply:
OK. I'll modifiy.
And modify english in README.md.